app/ocr.py

```diff
@@ -509,25 +509,63 @@ def references_LEA27_581(self) -> bool:
 
     def get_gender(self) -> str:
         """
-        • This field needs to be validated. Currently, it assumes the
-        sex of the seeker by the number of instances of pronouns in the
-        document.
-        """
-        male = 0
-        female = 0
-
-        for token in self.doc:
-            if token.text in {'him', 'his', 'he'}:
-                male += 1
-            elif token.text in {'her', 'hers', 'she'}:
-                female += 1
-
-        if male > female:
-            return 'male'
-        elif female > male:
-            return 'female'
+        Based on sentences where the Respondent/Applicant keywords are found,
+        count the instances of gendered pronouns. This approach assumes the
+        sentence refers to subject in shorthand by using gendered pronouns
+        as opposed to keywords multiple times in sentence. This function
+        returns a final result to be packaged into json.
+        Parameters:
+        spacy.doc (obj):
+        Returns:
+        str: Gender
+        """
+
+        # Search terms formatted
+        phrases = ["Respondent's", "Respondent", "respondent's", "respondent", "respondents",

```

Comment:
One idea to consider in future for the `phrases` would be to use the lemma of the word to reduce the need for plural/possessive nouns.

Reply:
Important note @ren-curry! If there's time, I will make an attempt to implement lemma functionality; else, this would be a  good-start hurdle for the next cohort.

Reply:
> One idea to consider in future for the `phrases` would be to use the lemma of the word to reduce the need for plural/possessive nouns.

Just updated the function to use lemma as you had suggested. Thank you for the feedback.